.circleci/config.yml

```diff
@@ -436,6 +415,68 @@ jobs:
           path: /tmp/results.html
       - store_artifacts:
           path: /tmp/results.json
+
+  cloud-e2e-k8s-idpe:
+    machine:
+      # use Ubuntu 16 image as jsonnet requires libraries newer than in Ubuntu 14
+      image: ubuntu-1604:202007-01
+    # E2e tests deploy kind and are very resource intensive.
+    # 2xlarge is the largest resource class for "machine"; there is no 2xlarge+ like the docker runners.
+    resource_class: 2xlarge
+    working_directory: ~/
+    steps:
+      - checkout:
+          path: ./ui
+      - install_jsonnet_binary
+      - install_kind_binary
+      - install_kubectl_binary
+      - install_kubecfg_binary
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout_k8s_idpe
+      - checkout_idpe
+      - checkout_monitor_ci
+      - run:
+          name: Make test artifact directories
+          command: |
+            mkdir -p /tmp/artifacts
+            mkdir -p /tmp/cypress_artifacts
+      - run:
+          name: Copy over the testing variables
+          command: cp ./monitor-ci/env.testing.cloud ./monitor-ci/.env
+      - run: docker login -u=$QUAY_USER -p=$QUAY_PASS quay.io
+      - run:
+          name: Build ui-e2e image
+          command: |
+            cd ./monitor-ci && make build NODE=cypress
+      # TODO: enable this when we remove cloud-e2e job
+      # - when:
+      #     condition:
+      #       equal: [master, << pipeline.git.branch >>]
+      #     steps:
+      #       # The cypress image used within the ui-e2e image is a non-standard slim image and its definition and versions are
+      #       # maintained here: https://github.com/influxdata/cypress-slim
+      #       - run: docker tag local/cypress:latest quay.io/influxdb/ui-e2e:${CIRCLE_SHA1}
+      #       - run: docker tag quay.io/influxdb/ui-e2e:${CIRCLE_SHA1} quay.io/influxdb/ui-e2e:latest
+      #       - run: docker push quay.io/influxdb/ui-e2e:${CIRCLE_SHA1}
+      #       - run: docker push quay.io/influxdb/ui-e2e:latest
+      - run:
+          name: Run E2e test
+          command: cd ./idpe/ && export QUAY_CD_USER=$QUAY_USER && export QUAY_CD_PASSWORD=$QUAY_PASS && ./scripts/ci/cd-app-e2e-test.bash | tee /tmp/artifacts/e2e-test.log || true
```

Comment:
the intention of `|| true` here is to suppress all `exit 1` codes until we want to disable `cloud-e2e` and depend on `cloud-e2e-k8s-idpe`

Reply:
i dont understand this. we can make it fail for a week if we dont gate on it and tell people to ignore it... but it shouldn't fail anyways right?

Reply:
if it fails, it'll block their PRs. So I have to have the job "pass" while the step failed. But right, it shouldn't fail (unless the tests fail for a good reason)

Reply:
didn't realize we can control what jobs are "required" via github - removing this `|| true` nonsense